src/main/resources/schema/moderne-organizations.graphqls

```diff
@@ -1,4 +1,10 @@
 type Query {
+    """
+    Deprecated: will be replaced by allOrganizations query.
+    The list of organizations that a repository belongs to.
+    """
+    organizations(repository: RepositoryInput!): [Organization!]! @deprecated
```

Comment:
```suggestion
    organizations(repository: RepositoryInput!): [Organization!]! @deprecated(reason: "Because… reasons")
```

I think most schema validators will nudge you toward adding a `reason` attribute

Reply:
done!